packages/visual-editor/src/components/header/ExpandedHeader.tsx

```diff
@@ -139,6 +139,13 @@ const expandedHeaderSectionFields: Fields<ExpandedHeaderProps> = {
               },
             },
             defaultItemProps: defaultMainLink,
+            getItemSummary: (item, i) => {
+              const { i18n } = useTranslation();
+              return (
+                resolveComponentData(item.label, i18n.language) ||
+                pt("Link", "Link") + " " + ((i ?? 0) + 1)
```

Comment:
Technically this won't translate correctly though it's better than what we had which is no translation at all